src/components/TimelineEvent.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import './TimelineEvent.css';
 import Timestamp from './Timestamp';
 
-const TimelineEvent = () => {
-  // Fill in your code here
-  return;
-}
+const postTimestamp = (props) => {
```

Comment:
I like (I think?) that you pulled things out into consts to use later on in your code.